nbs/03_export2html.ipynb

```diff
@@ -1155,6 +1155,80 @@
     "    return pnb"
```

Comment:
<p>Since it's used as one of the processors, <code>cite2link</code> can be public  (otherwise people can't adjust <code>process_cell</code>  and <code>process_cells</code>  as they want).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>